adapters/__tests__/coingecko.spec.ts

```diff
@@ -0,0 +1,37 @@
+
+import nock from 'nock'
+import coingecko from '../src/coingecko'
+import BigNumber from 'bignumber.js'
+
+describe('multi price fetch', () => {
```

Comment:
should combine this with coingecko.e2e.ts and just call it` __tests__/coingecko.test.ts`

Reply:
Separate into another PR?